packages/js/README.md

```diff
@@ -238,6 +238,39 @@ use `client.getIsRegistered` method
 client.getIsRegistered().then(isRegistered => {...})
 ```
 
+### Anonymous Login for AI assistant
+
+The TelnyxRTC client can be used with an AI assistant by using the `anonymous_login` method. This allows you to authenticate without a SIP connection, making it easier to integrate with AI assistants.
+
+```js
+const client = new TelnyxRTC({
+  anonymous_login: {
+    target_id: 'YOUR_TARGET_ID',
```

Comment:
I assume this is the `telephony_credentials` ID, right? If so it'd be good to leave a sentence explaining it here https://developers.telnyx.com/docs/voice/webrtc/js-sdk/interfaces/iclientoptions#anonymous_login